recipes-qt/qt5/qtnetworkauth_git.bb

```diff
@@ -4,6 +4,10 @@ LIC_FILES_CHKSUM = " \
     file://LICENSE.FDL;md5=6d9f2a9af4c8b8c3c769f6cc1b6aaf7e \
 "
 
+SRC_URI += "\
```

Comment:
`require qt5-git.inc` would overwrite SRC_URI so patch doesn't apply. This has to go below that.